docbook/Developers_Guide/en-US/Plugins_Building.xml

```diff
@@ -90,23 +101,216 @@ class ExamplePlugin extends MantisPlugin {
+											major release of MantisBT.
+											(i.e. for 2.x.y we would add
+											<literal>'&lt;2</literal>').
+										</para>
+										<para>This effectively disables plugins
+											which have not been specifically designed
+											for a new major Mantis release, thus
+											forcing authors to review their code,
+											adapt it if necessary, and release a
+											new version of the plugin with updated
+											dependencies.
+										</para>
+									</note>
+								</para>
+							</listitem>
+							<listitem>
+								<para><emphasis>Both minimum and maximum</emphasis>:
+									the two version numbers must be separated by
+									a comma.
+								</para>
+							</listitem>
+						</itemizedlist>
+						<para>
+							Here are a few examples to illustrate the above
+							explanations, assuming that the current Mantis
+							release (<emphasis>MantisCore</emphasis> version)
+							is 2.1:
+							<itemizedlist>
+								<listitem>
+									<para>Old release without a maximum version
+										specified
+										<programlisting>
+$this->requires = array( 'MantisCore' => '1.3.1' );
+										</programlisting>
+										The plugin is compatible with MantisBT
+										>= 1.3.1 and &lt; 2(.0) - note the maximum
+										version that was added by the system.
+									</para>
+								</listitem>
+								<listitem>
+									<para>Current release without a maximum
+										version specified
+										<programlisting>
+$this->requires = array( 'MantisCore' => '2.0' );
+										</programlisting>
+										The plugin is compatible with MantisBT
+										>= 2.0 and &lt; 3.0 (the latter is implicit);
+										code supporting older releases (e.g. 1.3)
+										must be maintained separately (i.e. in a
+										different branch).
+									</para>
+								</listitem>
+								<listitem>
+									<para>Only specify a maximum version
+										<programlisting>
+$this->requires = array( 'MantisCore' => '&lt; 3.1' );
+										</programlisting>
+										The plugin is compatible up to MantisBT
+										3.1 (not included).
+									</para>
+								</listitem>
+								<listitem>
+									<para>Old release with a maximum version
+										<programlisting>
+$this->requires = array( 'MantisCore' => '1.3, &lt; 4.0' );
+										</programlisting>
+										The plugin is compatible with MantisBT
+										>= 1.3 and &lt; 4.0.
+									</para>
+								</listitem>
+							</itemizedlist>
+						</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>uses</term>
+					<listitem>
+						<para>An array of key/value pairs of
+							basename/version optional (soft) plugin dependencies.
+							See <literal>requires</literal> above
+							for details on how to specify versions.
+						</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>author</term>
+					<listitem>
+						<para>Your name, or an array of names.</para>
```

Comment:
Currently if an array of names is provided, the code collapses it into a comma-delimited string, so that's pretty much the same thing, but I believe keeping the array makes sense 
1. for backwards-compatibility, and
2. if in the future we decide to allow specifying additional details about the authors (e.g. individual e-mails address), like [Composer does](https://getcomposer.org/doc/04-schema.md#authors)